setup.c

```diff
@@ -42,7 +42,7 @@ void setup(void)
 
 	// enable RNG
 	rcc_periph_clock_enable(RCC_RNG);
-	RNG_CR |= RNG_CR_IE | RNG_CR_RNGEN;
+	RNG_CR |= RNG_CR_RNGEN;
```

Comment:
I think we might want to add `RNG_CR &= ~RNG_CR_IE;` to `setupApp()` function so we have the new behaviour also when using old bootloader and newer firmware, right?